proof/invariant-abstract/AARCH64/ArchRetype_AI.thy

```diff
@@ -125,10 +125,7 @@ declare post_retype_invs_check_def[simp]
 
 end
 
-
-context begin interpretation Arch .
-requalify_consts post_retype_invs_check
-end
+arch_requalify_consts post_retype_invs_check
```

Comment:
Requalification in an Arch file

Reply:
Unavoidable: const introduced in Arch locale that needs to be referenced generically in this theory.